src/Data.js

```diff
@@ -1,5 +1,7 @@
-let base = 'https://task-manager-by.herokuapp.com';
+// let base = 'https://task-manager-by.herokuapp.com';
+let base = 'http://localhost:8888';
 
+export let urlWebSocket = 'ws://localhost:8080';
```

Comment:
I don't think it'll work on production

Reply:
We don't have stable work WS on pordaction. 

I am running the backend locally

Reply:
why it's not stable? did you add some code to backend?

Reply:
not. The web socket did not work for us. And you told me to run on the backend